openapi.yaml

```
@@ -786,6 +802,8 @@ components:
               enum:
                 - DAYS_BEFORE
     cq-before-or-same:
+      description: "Used within CONCEPT_QUERY to describe relative temporal relationships between complex events.
+            The result of `preceding` has be at or before `index`' sampled day."
```

```suggestion
            The result of `preceding` has to be at or before the sampled day's `index`."
```
Ich denke, *the sampled day's `index`* liest sich am besten

---

Ist aber leider nicht richtig, index ist eine Anfrage(Ergebnis) und von dem wird ein Tag gesampled